### PR TITLE
MODEL_COMPARISION AND RESULTS

### PR DESCRIPTION
**Pull request by spursbyte**
this is in lieu with issue no #14. kindly review the notebook and the readme file.

### CODE REVIEW

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
@spursbyte You made a lot of mistakes in this PR
1. You send the PR to the base branch but you need to send it to the New_Pipeline branch
2. You uploaded the Notebook file. But we don't need a notebook file I already intimated to develop two different scripts.
Follow up the above mistakes then we will intimate further more.

**Comment by spursbyte** [comment]
I HAVE UPDATED THE REQUIRED FILES AND DELETED THE NOTEBOOK. I HAVE UPDATED THE README FOR MY PR ACCORDINGLY KINDLY REVIEW.

File: breast_cancer_model_exploration/breast_cancer_ann.py [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# Importing libraries
```
Comment (srimani-programmer) [inline breast_cancer_model_exploration/breast_cancer_ann.py:1]: @spursbyte Add your models file into the models folder which is available in the project.

File: breast_cancer_model_exploration/gridsearch.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+import numpy as np
```
Comment (srimani-programmer) [inline breast_cancer_model_exploration/gridsearch.py:1]: Add this one to the models folder.

**Review by srimani-programmer — CHANGES_REQUESTED** [review]
Do the requested changes.

**Comment by spursbyte** [comment]
done with the cahnges kindly review..one more thing i have also the readme file of the model in the model directory should i keep it or delete it?

File: Model/gridsearch.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    end = time.time()
+    print("Run Time: %f" % (end - start))
+    return model_svc
+#predicting ion svc
+def svc_model_predict(model_svc,X_test,y_test):
+    predictions = model_svc.predict(X_test)
+    print("Accuracy score %f" % accuracy_score(y_test, predictions))
+    print(confusion_matrix(y_test, predictions))
+    return predictions
+
```
Comment (srimani-programmer) [inline Model/gridsearch.py:83]: @spursbyte remove extra line just add one it's enough

File: Model/breast_cancer_ann.py [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+    cm = confusion_matrix(y_test, y_pred)
+    print(cm)
+    accuracy = (cm[0][0] + cm[1][1]) / (cm[0][0] + cm[0][1] + cm[1][0] + cm[1][1])
+    print("Accuracy: " + str(accuracy * 100) + "%")
+
+#saving the model in .h5 format
+def ann_model_save(classifier):
+    classifier.save('breast_cancer_model.h5')
+
+
```
Comment (srimani-programmer) [inline Model/breast_cancer_ann.py:67]: @spursbyte remove extra line.

**Review by srimani-programmer — CHANGES_REQUESTED** [review]

File: Model/breast_cancer_ann.py (hunk 4 above)
Comment (spursbyte) [reply Model/breast_cancer_ann.py:67]: Ok on it

Comment (spursbyte) [reply Model/breast_cancer_ann.py:65]: @srimani-programmer  DONE WITH TTHE CHANGES

File: Model/readme.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+### VARIOUS CLASSIFICATION MODEL COMPARISION AND CHOOSING THE BEST ONE.
```
[the remark this answers is not in the document]
Comment (spursbyte) [reply Model/readme.md]: DONE WITH THE CHANGES

File: Model/gridsearch.py (hunk 3 above)
Comment (spursbyte) [reply Model/gridsearch.py:83]: DIME WITH THE CHANGES

**Review by srimani-programmer — APPROVED** [review]

**Pull request merged**